components/edit-team-form.js

```diff
@@ -1,9 +1,31 @@
 import React from 'react'
 import { Formik, Field, Form } from 'formik'
+import urlRegex from 'url-regex'
 import Button from '../components/button'
 import dynamic from 'next/dynamic'
 const FormMap = dynamic(() => import('../components/form-map'), { ssr: false })
 
+const isUrl = urlRegex({ exact: true })
+
+function validateUrl (value) {
+  if (!isUrl.test(value)) return 'Please enter a valid url'
```

Comment:
```suggestion
  if (value && !isUrl.test(value)) return 'Please enter a valid url'
```
Validation should only occur if user enters text into the input, allowing the URL to be optional.

Reply:
Yesss, thank you